solidity/contracts/isms/routing/DomainRoutingIsm.sol

```diff
@@ -74,6 +70,37 @@ contract DomainRoutingIsm is
         _set(_domain, address(_module));
     }
 
+    function set(
+        uint32[] calldata _domains,
+        IInterchainSecurityModule[] calldata __modules
+    ) external onlyOwner {
+        _set(_domains, __modules);
+    }
+
+    /**
+     * @notice Adds the specified origin domain
+     * @dev Reverts if the domain already exists
+     * @dev Behavior similar to set but useful for distinguishing by selector
+     * @param _domain The origin domain
+     * @param _module The ISM to use to verify messages
+     */
+    function add(
+        uint32 _domain,
+        IInterchainSecurityModule _module
+    ) external onlyOwner {
+        _add(_domain, address(_module));
+    }
+
+    function add(
```

Comment:
Do we want to use structs or arrays in configuration functions like this? You recommended structs elsewhere: https://github.com/hyperlane-xyz/hyperlane-monorepo/pull/6625#discussion_r2180819285